app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
-  validates :name, :email, :google_token, :google_refresh_token, presence: true
```

Comment:
Double checking... did we inadvertently delete the google token?

Reply:
Nope! @markcyen we 'advertently' deleted this validation because it was causing issues on the FE for Zach.